osu.Game/Screens/Menu/MainMenu.cs

```diff
@@ -44,6 +46,16 @@ public class MainMenu : OsuScreen
 
         protected override BackgroundScreen CreateBackground() => background;
 
+        protected override RichPresence Presence => new RichPresence
+        {
+            Details = "Idling",
+            Assets = new Assets()
```

Comment:
This cannot be repeated each usage. Making a base implementation in OsuScreen and only changing what is required is one fix but likely not what we want either.